tests/Unit/ImageComparatorTest.php

```diff
@@ -272,6 +272,24 @@ public function testCompareBlurredImages(): void
         $this->assertGreaterThan(65.00, $result);
     }
 
+    public function testCompare1(): void
```

Comment:
```suggestion
    public function testCompareSolildColors($image1, $image2): void
```

You can also use dataProviders and pass needed colors as function parameters. You can see DataProviders examples in other tests